src/components/SlidePanel/SlidePanel.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import "./_slide-panel.scss";
+
+function SlidePanel({ children, onClose }) {
+  return (
+    <div className="slide-panel">
+      <div className="slide-panel--close">
```

Comment:
Please use a button similar to the model pattern:
```html
<button class="p-modal__close" aria-label="Close active modal" aria-controls="modal">Close</button>
```
Then you want need to worry about `role`, `tabIndex` or the cursor styling.

Reply:
Good call, done